kubernetes/samples/scripts/create-rcu-schema/README.md

```diff
@@ -1,56 +1,21 @@
-# Creating an Oracle database service and RCU schema for a Fusion Middleware domain
+# Managing RCU schema for a Fusion Middleware domain
 
 The sample scripts in this directory demonstrate how to:
-* Start an Oracle database (DB) service in a Kubernetes cluster.
-* Stop an Oracle DB service in a Kubernetes cluster.
 * Create an RCU schema in the Oracle DB that will be used by a Fusion Middleware domain.
 * Delete the RCU schema in the Oracle DB used by a Fusion Middleware domain.
 
-## Start an Oracle database service in a Kubernetes cluster
+## Start an Oracle Database service in a Kubernetes cluster
 
-Use this script to create an Oracle DB service in the default namespace, with the default credentials, in the Oracle Database Slim image.
+Use the script samples/scripts/create-oracle-db-service/start-db-service.sh
```

Comment:
file path to be inside back ticks ``.
Execute the script `samples/scripts/create-oracle-db-service/start-db-service.sh` to create an Oracle DB service.

Reply:
done